stairway/src/main/java/bio/terra/stairway/FlightMap.java

```diff
@@ -95,13 +95,26 @@ boolean isEmpty() {
   }
 
   /**
-   * Return the object from the hash map deserialized to the right type. Throw an exception if the
-   * Object cannot be deserialized to that type.
+   * Returns true if this map contains a mapping of the passed key.
+   *
+   * @param key to look up in the map
+   * @return true if present
+   */
+  boolean containsKey(String key) {
+    return map.containsKey(key);
+  }
+
+  /**
+   * Return the object from the hash map deserialized to the right non-parameterized type. Throw an
```

Comment:
Personally I would avoid the term `hash map` here as it's an implementation detail that the caller doesn't need to know about. If we changed `FlightMap` to use a different underlying data structure it shouldn't affect the api behavior. I would say something like `Given a key, return the flight map object for that key.`

Reply:
Good suggestion, replaced throughout.